src/getting-started.md

```diff
@@ -33,71 +33,65 @@ instead.
+remembered the answer, and Anki will choose when to show you the card again. If, for example, Anki decides to show you the card again in 30 days, we say the card has a 30 day interval.
 
 #### Types of Cards
 
-- **New:** A new card is one that you have downloaded or entered in, but have never studied before.
+- **New:** A new card is one that you have downloaded or created yourself, but have never studied before.
 
 - **Learning:** Cards that were seen for the first time recently, and are still being learnt.
 
-- **Review:** Cards that were previously learnt, and now need to be reviewed so you don’t forget them.
+- **Review:** Cards that were previously learnt, and now need to be timely reviewed so you don’t forget them.
   There are two types of review cards:
-    - **Young:** A young card is one that has an interval of less than 21 days, but is
-   not in learning.
+    - **Young:** A young card is one that has an interval of less than 21 days.
     - **Mature:** A mature card is one that has an interval of 21 days or greater.
 
 - **Relearn:** A relearning card is a card that you have failed in review mode, thus
   returning it to learning mode to be relearned.
 
 ### Decks
 
-A 'deck' is a group of cards. You can place cards in different decks to
+A _deck_ is a group of cards. You can place cards in different decks to
 study parts of your card collection instead of studying everything at
 once. Each deck can have different settings, such as how many new cards
 to show each day, or how long to wait until cards are shown again.
 
 Decks can contain other decks, which allows you to organize decks into a
-tree. Anki uses “::” to show different levels. A deck called
-“Chinese::Hanzi” refers to a “Hanzi” deck, which is part of a “Chinese”
-deck. If you select “Hanzi” then only the Hanzi cards will be shown; if
-you select “Chinese” then all Chinese cards, including Hanzi cards, will
-be shown.
-
-To place decks into a tree, you can either name them with “::” between
-each level, or drag and drop them from the deck list. Decks that have
-been nested under another deck (that is, that have at least one “::” in
-their names) are often called 'subdecks', and top-level decks are
-sometimes called 'superdecks' or 'parent decks'.
-
-Anki starts with a deck called “default”; any cards which have somehow
+tree. Anki uses double colons ("::") to show different levels within the deck tree. For example, a deck called
+"Chinese::Hanzi" refers to a "Hanzi" deck, which is part of a "Chinese"
+deck. If you select "Hanzi", then only the Hanzi cards will be shown; if
+you select "Chinese", then all the Chinese cards will be shown, including the Hanzi cards.
+
+To place decks under a tree, you can either name them with double colons between
+each level, or drag and drop them within the deck list. Decks that have
+been placed under a tree are often called "subdecks", and top-level decks are called "parent decks".
```

Comment:
How about "within" instead of "into"? I'll apply this if no qualms.

```suggestion
To place decks within a tree, you can either name them with double colons between
each level, or drag and drop them within the deck list. Decks that have
been placed inside another deck are often called "subdecks", and top-level decks are called "parent decks".
```